SingularityUI/app/components/machines/ModalButton.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { PropTypes } from 'react';
+import FormModal from '../common/FormModal';
+import {OverlayTrigger, Tooltip} from 'react-bootstrap/lib';
+
+class ModalButton extends React.Component {
```

Comment:
I've been using this pattern for [Modal](https://github.com/HubSpot/Singularity/blob/8ee68b81edb0cdc749e03f9de46ddca91798d3f4/SingularityUI/app/components/requests/BounceModal.jsx)s and [Button](https://github.com/HubSpot/Singularity/blob/8ee68b81edb0cdc749e03f9de46ddca91798d3f4/SingularityUI/app/components/requests/BounceButton.jsx)s